src/tests/ftest/pool/bad_evict.py

```diff
@@ -78,41 +65,35 @@ def test_evict(self):
                 break
 
         saveduuid = None
-        savedgroup = None
         pool = None
 
         try:
             # initialize a python pool object then create the underlying
             # daos storage
-            pool = DaosPool(self.context)
-            pool.create(createmode, createuid, creategid,
-                        createsize, createsetid, None)
-
-            # trash the pool group value
-            savedgroup = pool.group
-            if evictset is None:
-                pool.group = None
-            else:
-                pool.set_group(evictset)
+            pool = TestPool(self.context, self.get_dmg_command())
+            pool.get_params(self)
+            pool.create()
 
             # trash the UUID value in various ways
             if excludeuuid is None:
                 saveduuid = (ctypes.c_ubyte * 16)(0)
                 for item in range(0, len(saveduuid)):
-                    saveduuid[item] = pool.uuid[item]
-                pool.uuid[0:] = [0 for item in range(0, len(pool.uuid))]
+                    saveduuid[item] = pool.pool.uuid[item]
+                pool.pool.uuid[0:] = \
+                    [0 for item in range(0, len(pool.pool.uuid))]
             elif excludeuuid == 'JUNK':
                 saveduuid = (ctypes.c_ubyte * 16)(0)
                 for item in range(0, len(saveduuid)):
-                    saveduuid[item] = pool.uuid[item]
-                pool.uuid[4] = 244
+                    saveduuid[item] = pool.pool.uuid[item]
+                pool.pool.uuid[4] = 244
 
-            pool.evict()
+            # evict the pool
+            self.get_dmg_command().pool_evict(pool=pool.pool.get_uuid_str())
```

Comment:
Can we use the evict method?

`pool.pool.evict()`

Reply:
Sure. Done.